core/mondoo-linux-workstation-security.mql.yaml

```diff
@@ -4,7 +4,7 @@
 policies:
   - uid: mondoo-linux-workstation-security
     name: Linux Workstation Security
-    version: 0.3.0
+    version: 1.0.0
```

Comment:
This should be 0.4.0 not 1.0.0

Reply:
Chris confirmed this change was intentional.  Approving.

Reply:
We want to bump to 1.0.0 since it more stable than a 0.x version